### DISCUSSION TITLE
记一个折磨了我一天半的 Bug — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#219] [Announcements]
https://mazhuang.org/2019/05/25/a-stupid-bug/ 

 最近开始学习后台开发，虽然与我以前从事的 Android 开发一样都是使用 Java 语言，但是技术栈完全不同，有太多的必备的「新」概念要去学习，而在对它们，以及别人写的代码有充分的了解之前，就可能会遇上这种一杯茶，一根烟，一个 Bug 一天根本改不完的情况。

**Comment by ZPJ8** [comment]
不错哦

**Comment by LeeYouth** [comment]
test

**Comment by huanrenwang** [comment]
顶一下

**Comment by aturX** [comment]
这坑用kafka的时候踩过

**Comment by Jia-Jianpeng** [comment]
test

**Comment by alex123-star** [comment]
11

**Comment by mzlogin** [comment]
> @yuchao-wang
> 哈哈，没想到你也是Android转Java后端的啊？我也是，18年转的。加个好友啊 18710083587

申请加你了。

**Comment by arctan95** [comment]
ok
